Tests/pairwise2_testCases.py

```diff
@@ -77,6 +77,23 @@ def test_warnings(self):
             self.assertIn("should not", str(w[-1].message))
 
 
+class TestPairwiseKeywordUsage(unittest.TestCase):
+    """Tests for keyword usage."""
+
+    def test_keywords(self):
+        """Test equality of calls with and without keywords."""
+        aligns = pairwise2.align.globalxx("GAACT", "GAT")
+        aligns_kw = pairwise2.align.globalxx(sequenceA="GAACT",
+                                             sequenceB="GAT")
```

Comment:
I think that would only be 80 characters as one line, well within the black limit of 88.

Reply:
OK